packages/e2e-tests/specs/writing-flow.test.js

```diff
@@ -141,6 +142,42 @@ describe( 'adding blocks', () => {
 		expect( await getEditedPostContent() ).toMatchSnapshot();
 	} );
 
+	it( 'should navigate around nested inline boundaries', async () => {
+		await setPostContent( `
+			<!-- wp:paragraph -->
+			<p><strong><em>1</em> <em>2</em></strong></p>
+			<!-- /wp:paragraph -->
+		` );
+
+		await page.click( '[data-type="core/paragraph"] [contenteditable]' );
```

Comment:
The way we enter this paragraph and assume some caret position seems fragile to me. What guarantee do we have that the click here occurs at some specific reliable point? 

I'd wonder if instead we could do one of:

- Craft the content manually before running your test interactions
- Otherwise ensure we're where we want to be in the text field (press "End" maybe if we're trying to go to the end of the field?)

Reply:
Done :)